## Drill Problems
### *Chapter 4 Section 8* -- AC Steady State Analysis

In [1]:
import numpy as np

#### Problem 1
1. Sketch the phasor diagram for a three-phase 3-wire unbalanced $Y-Y$ system: $Z_1(j\omega)=\infty$, $Z_2(j\omega)=Z_3(j\omega)=100\Omega + j100\Omega$.

$$ Z_2(j\omega) = Z_3(j\omega) = 100\Omega(1+j) = 100\sqrt{2}\Omega\,\angle45^\circ \quad\quad\text{and}\quad\quad J_\mathrm{i}(j\omega) = I_\mathrm{i}(j\omega) \;\forall\,\mathrm{i}\in\{1,2,3\}$$

Via Ohm's Law in the frequency domain,  

$$ V_\mathrm{i}(j\omega) = Z_\mathrm{i}(j\omega)I_\mathrm{i}(j\omega)\quad\forall\,\mathrm{i}\in\{1,2,3\}$$

Via KVL,  

$$ E_1(j\omega) - E_2(j\omega) = V_1(j\omega) - V_2(j\omega) = U_{12}(j\omega) \quad\quad\text{and}\quad\quad E_2(j\omega) - E_3(j\omega) = V_2(j\omega) - V_3(j\omega) = U_{23}(j\omega) $$

Via the above two equations and equation (5b,c) and equivalence principle,  

$$ V_1(j\omega) - V_2(j\omega) = \sqrt{3}E\angle30^\circ\quad\quad\text{and}\quad\quad V_2(j\omega)-V_3(j\omega) = \sqrt{3}E\angle270^\circ$$

Via the above two equations and Ohm's Law,  

$$ Z_1(j\omega)I_1(j\omega) - Z_2(j\omega)I_2(j\omega) = \sqrt{3}E\angle30^\circ \quad\quad\text{and}\quad\quad Z_2(j\omega)\big(I_2(j\omega) - I_3(j\omega)\big) = \sqrt{3}E\angle270^\circ$$

Via KCL,  

$$ I_1(j\omega) = -\big(I_2(j\omega) + I_3(j\omega)\big)$$

Via the above three equations and subsitution,  

\begin{equation}\tag{1.1a}
    Z_1(j\omega)\big(I_2(j\omega) + I_3(j\omega)\big)  + Z_2(j\omega)I_2(j\omega) = \sqrt{3}E\angle210^\circ
\end{equation}

\begin{equation}\tag{1.1b}
    Z_2(j\omega)\big(I_2(j\omega) - I_3(j\omega)\big) = \sqrt{3}E\angle270^\circ
\end{equation}

Via equation (1.1a),   

\begin{equation}\tag{1.2}
    I_3(j\omega) = -I_2(j\omega)
\end{equation}

And, via equations (1.2) and (1.1b),  

\begin{equation}\tag{1.3}
    I_2(j\omega) = \frac{\sqrt{3}E\angle270^\circ}{2Z_2(j\omega)} = \frac{\sqrt{3}E}{200\sqrt{2}\Omega}\,\angle225^\circ
\end{equation}

Via equations (1.3), (1.2), and KCL,  

\begin{equation}\tag{1.4}
    \text{a: }\; I_3(j\omega) = \frac{\sqrt{3}E}{200\sqrt{2}\Omega}\,\angle45^\circ\quad\quad\text{and}\quad\quad\text{b: }\;I_1(j\omega) = 0
\end{equation}

Finally, 

\begin{equation}\tag{1.5}
    V_1(j\omega) = 0\quad\quad V_2(j\omega) = 0.5\sqrt{3}E\angle225^\circ\quad\quad V_3(j\omega) = 0.5\sqrt{3}E\angle45^\circ
\end{equation}

#### Problem 2
2. The total power of a three-phase balanced wye load is $6 \mathrm{kW}$. What is the power factor of each phase load, if the line voltage is $400 \mathrm{V}$ and the line current is $10 \mathrm{A}$?

$$ P = 6\mathrm{kW} = 3VI\mathrm{pf}\quad\quad U(\omega) = U = \sqrt{3}V= 400\mathrm{V}\quad\quad J=I=10\mathrm{A}$$

$$ V = \frac{400\mathrm{V}}{\sqrt{3}} \quad\quad \mathrm{pf} = \frac{P}{3VI} = \frac{P}{400\sqrt{3}\mathrm{V}\cdot I}$$

$$ \mathrm{pf} = \frac{600}{400\sqrt{3}} = \frac{\sqrt{3}}{2}$$

#### Problem 3
3. A balanced delta load with phase impedance of $10\Omega\angle30^\circ$ is connected to $230 \mathrm{V}$ lines.  Determine the total real power supplied.

$$ P = 3\cdot \frac{U^2}{10\Omega\cos30^\circ} = \frac{\sqrt{3}U^2}{5\Omega} = \sqrt{3}\cdot 10.58\mathrm{kW}$$

In [2]:
np.cos(np.radians(30))

0.8660254037844387

In [3]:
0.5*np.sqrt(3)

0.8660254037844386

In [4]:
(230**2)/5

10580.0

#### Problem 4
4. A balanced wye load with phase impedance of $100\Omega$ is connected to $230 \mathrm{V}$ lines. Determine the total real power delivered to the load.

$$ P = 3\frac{V^2}{R} \quad\quad U = \sqrt{3}V = 230\mathrm{V} \quad\quad Z(j\omega) = Z= R=100\Omega$$

$$ V = \frac{230\mathrm{V}}{\sqrt{3}} \quad\implies\quad P = \frac{230^2\mathrm{V}^2}{100\Omega} = 529\mathrm{W}$$

In [5]:
(230**2)/100

529.0

#### Problem 5
5. For the balanced wye-delta system: $E=100\mathrm{V}$, $Z(j\omega)=100\Omega$, find RMS value of a line current.

$$ V = U = \sqrt{3}E = \sqrt{3}100\mathrm{V} \quad\quad\text{and}\quad\quad I = \frac{V}{Z} = \sqrt{3}\mathrm{A}$$

$$ J = \sqrt{3}I = 3\mathrm{A}$$

#### Problem 7
7. For the three-phase 4-wire balanced system: $E=100\mathrm{V}$ and $Z(j\omega)=Z=R=100\Omega$, find the total power supplied after opening phase 1, draw the phasor diagram.

$$ P = 2\frac{E^2}{R} = 200\mathrm{W}$$

$$ V_1(j\omega) = 0 \quad\quad I_1(j\omega) = 0 \quad\quad V_\mathrm{i}(j\omega)=E_\mathrm{i}(j\omega)\;\forall\,\mathrm{i}\in\{2,3\}$$

$$ I_2(j\omega) = 1\mathrm{A}\,\angle240^\circ\quad\quad\text{and}\quad\quad I_3(j\omega) = 1\mathrm{A}\,\angle120^\circ$$

#### Problem 8
8. For the three-phase 3-wire balanced system in Y-Y configuration: $E=100\mathrm{V}$ and $Z(j\omega)=100\Omega$, find the total power supplied after shorting phase 1, draw the phasor diagram.

\begin{equation}\tag{8.1}
    P = \frac{(V_2)^2 + (V_3)^2}{R}
\end{equation}

\begin{equation}\tag{8.2}
    V_2(j\omega) = U_{21}(j\omega) = E_2(j\omega) - E_1(j\omega) = \sqrt{3}E\angle210^\circ
\end{equation}

$$ U_{23}(j\omega) = V_2(j\omega) - V_3(j\omega) = \sqrt{3}E\angle270^\circ$$

$$ V_3(j\omega) = \sqrt{3}E\angle210^\circ - \sqrt{3}E\angle270^\circ = \sqrt{3}E\big(\cos 210^\circ - \cos 270^\circ + j\sin210^\circ - j\sin270^\circ\big)$$

$$ V_3(j\omega)= \sqrt{3}E\big(0.5\sqrt{3} + j0.5\big) = 0.5\sqrt{3}E\big(\sqrt{3} + j\big)$$

\begin{equation}\tag{8.3}
    V_3(\omega) = V_3 = \sqrt{3}E
\end{equation}

Finally,  

\begin{equation}\tag{8.4}
    P = 6\,\frac{E^2}{R} = 600\mathrm{W}
\end{equation}

In [6]:
np.cos(np.radians(210))

-0.8660254037844386

In [7]:
np.sin(np.radians(210))

-0.5000000000000001

#### Problem 9
9. Find the total power supplied: $E=100\mathrm{V}$ and $R=100\Omega$.  

![Fig-P.4.8.9](Figures/Fig-P.4.8.9.png)

$$ P = 4\frac{E^2}{R} = 400\mathrm{W}$$

#### Problem 10
10. Three adjacent houses take electricity from the three successive phases of a $230 \mathrm{V}$ supply. Find the resultant neutral current when they consume, in order of phase sequence, $ 1 \mathrm{A}$ at unity power factor, $2 \mathrm{A}$ at $0.75$ lagging and $2\mathrm{A}$ at $0.75$ leading. What is the total power absorbed ?

In [10]:
I_1 = 1; I_2 = I_3=2

Via the fact that there is a nuetral line and KVL,  

$$ V_\mathrm{i}(j\omega) = E_\mathrm{i}(j\omega)\quad\forall\,\mathrm{i}\in\{1,2,3\}$$

Which implies that  

\begin{equation}\tag{10.1}
    V = E = 230\mathrm{V}
\end{equation}

Note that each house drops a different phase voltage $V_\mathrm{i}(j\omega)$ with load neutral as the reference point.

Recall that, via KCL:  

$$ I_0(j\omega) = \sum_{\mathrm{i}=1}^3 I_\mathrm{i}(j\omega)$$

The magnitude of the impedance of each load (house) is:  

$$ Z_1 = \frac{E}{1\mathrm{A}} = 230\Omega \quad\quad Z_2 = \frac{E}{2\mathrm{A}} = 115\Omega \quad\quad Z_3 = \frac{E}{2\mathrm{A}} = 115\Omega$$

And, the direction of each impedance is given by:  

$$ \varphi_1 = 0 \quad\quad \varphi_2 = \arccos 0.75\approxeq 41.41^\circ \quad\quad\varphi_3 = -\arccos0.75\approxeq -41.41^\circ$$

In [9]:
varphi_2 = np.degrees(np.arccos(0.75))
varphi_3 = -varphi_2

The phase angle of the phase currrents is:  

$$ \alpha_{i1} = 0 \quad\quad \alpha_{i2} = 240^\circ - \varphi_2 \quad\quad \alpha_{i3} = 120^\circ - \varphi_3$$

In [11]:
alpha_i2 = 240 - varphi_2; alpha_i3 = 120-varphi_3

Thus, the neutral current is:  

$$ I_0(j\omega) = 1\mathrm{A}\angle0 + 2\mathrm{A}\angle\alpha_{i2} + 2\mathrm{A}\angle\alpha_{i3} = 1\mathrm{A} + 2\mathrm{A}\cos\alpha_{i2} + 2\mathrm{A}\cos\alpha_{i3} + j2\mathrm{A}\sin\alpha_{i2} + j2\mathrm{A}\sin\alpha_{i3}$$

In [12]:
I_0 = complex(1 + 2*np.cos(np.radians(alpha_i2)) + 2*np.cos(np.radians(alpha_i3)), \
              2*np.sin(np.radians(alpha_i2)) + 2*np.sin(np.radians(alpha_i3)))
I_0

(-2.7912878474779195+4.440892098500626e-16j)

\begin{equation}\tag{10.2}
    I_0(j\omega) \approxeq -2.79 \mathrm{A}
\end{equation}

The total power is given by:  

$$ P = \frac{V^2}{R_1} + \frac{V^2}{R_2} + \frac{V^2}{R_3}$$

The resistances of each house (load) are given by:  

$$ R_1 = 230\Omega\quad\quad R_2 = R_3 = 0.75Z_2 = 86.25\Omega$$

In [13]:
115*0.75

86.25

Finally,  

$$ P = 1456.666\dots\mathrm{W}$$

In [15]:
2*(230**2)/86.25 + 230

1456.6666666666667